src/components/AccordionItem/AccordionItem.js

```diff
@@ -133,12 +135,20 @@ export default class AccordionItem extends Component {
           aria-expanded={this.state.open}
           className={`${prefix}--accordion__heading`}
           onClick={this.handleHeadingClick}>
-          <Icon
-            className={`${prefix}--accordion__arrow`}
-            icon={iconChevronRight}
-            description={iconDescription}
-            role={null} // eslint-disable-line jsx-a11y/aria-role
-          />
+          {componentsX ? (
+            <ChevronRight16
+              className={`${prefix}--accordion__arrow`}
+              icon={iconChevronRight}
```

Comment:
I don't think the `icon prop` is needed for this. View example here: https://ibm.github.io/carbon-elements/icons-react/examples/storybook/?selectedKind=ChevronRight16&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Fstories%2Fstories-panel

Reply:
Though it still displays properly with it.

Reply:
Ah, well spotted! Have corrected this now. Cheers.